common/common_test.go

```diff
@@ -881,3 +882,54 @@ func TestGenerateRandomString(t *testing.T) {
 		t.Error("GenerateRandomString() unexpected test validation result")
 	}
 }
+
```

Comment:
The test `TestErrors` now fails in this PR:
`    common_test.go:757: targets not achieved`

Reply:
Tried to work out how TestErrors ended up back in here. `git-reflog` and `git-stash` show no signs of the change.
I definitely gutted it and moved TestMultiError into there, so I must have not got as far as committing it for some reason.
Everything I moved into TestMultiError is definitely covering it all as I remember.

Oh well.

Fixed and pushed